clients/fides-js/src/fides-tcf.ts

```diff
@@ -281,6 +283,13 @@ const _Fides: FidesGlobal = {
       // Nothing to show if there's no experience
       return false;
     }
+    if (
+      this.options.fidesKnownPreference === ConsentMethod.ACCEPT ||
+      this.options.fidesKnownPreference === ConsentMethod.REJECT
```

Comment:
Can we extract this into a `isOverrideAcceptOrReject` var or something similar that can be shared with non-tcf version too?